src/filter.js

```diff
@@ -99,6 +99,9 @@ define(function (require) {
     if (type) {
       this.setType(type);
     }
+
+    this._on = true;
+    this.type = this.biquad.type;
```

Comment:
At first glance, it wasn't clear why there is a duplicate property. Now I see how this is useful for `toggle()`. Maybe rename to `this._untoggledType` to indicate that it's a private property and describe how it is used?

What happens if a user calls `setType()` on the filter itself, and then `toggle`? It'd be good to write a test case for this and make sure it can pass. [Getters and setters](https://javascriptplayground.com/blog/2013/12/es5-getters-setters/) might be useful here.